.changeset/olive-hats-invent.md

```diff
@@ -0,0 +1,11 @@
+---
+'@leafygreen-ui/table': minor
+---
+
+Introduces a new prop, `overrideTruncation`, on `<Cell />` to allow overriding truncation for individual cells. When `shouldTruncate` is set to true on `<Table />`, `overflow: hidden` is applied to all cells, which can unintentionally clip hover and focus states on LeafyGreen components like `<Button />`. Setting `overrideTruncation` to true removes the overflow styles for that specific cell, ensuring interactive elements display correctly.
```

Comment:
I went with a prop that hopefully conveys that it will only turn off truncation for a cell if `shouldTruncate` is `true`.

I considered `overrideShouldTruncate` but i felt that it conveyed that prop this would always be the invert of `shouldTruncate`.

I'm open to other prop names or solutions.

Reply:
This is so much better than my solution. Thank you 🙏🏽 And I also agree with your first comment; It's a code smell.